Analysis/Stan/mapping_model_inference.stan

```diff
@@ -123,28 +176,39 @@ parameters {
   
   // Spatial random effects
   real <lower=0, upper=1> rho;    // spatial correlation parameter
-  real log_std_dev_w;             // precision of the spatial effects
+  real<lower=0> std_dev_w;             // precision of the spatial effects
   vector[smooth_grid_N] w;        // spatial random effect
   
   // Temporal random effects
-  vector[T*do_time_slice_effect] eta_tilde;    // uncentered temporal random effects
-  real <lower=0> sigma_eta[do_time_slice_effect*do_infer_sd_eta];    // sd of temporal random effects
+  vector[size_eta] eta_tilde;    // uncentered temporal random effects
+  real <lower=0> sigma_eta[size_sd_eta];    // sd of temporal random effects
   
   // Covariate effects
   vector[ncovar] betas;
   
+  // Overdispersion parameters
+  vector<lower=0>[N_admin_lev*do_overdispersion] inv_od_param;
+  
 }
 
 transformed parameters {
   
   vector[T*do_time_slice_effect] eta;    // temporal random effects
   vector[M] modeled_cases;        // expected number of cases for each observation
-  real std_dev_w = exp(log_std_dev_w);    // sd of spatial random effects
+  // real std_dev_w = exp(log_std_dev_w);    // sd of spatial random effects
   vector[N] grid_cases;       // cases modeled in each gridcell and time point
   real previous_debugs = 0;
   real sigma_eta_val;        // value of sigma_eta. This is either fixed to sigma_eta_scale if do_infer_sd_eta==0, or sigma_eta otherwise
+  vector[N_admin_lev*do_overdispersion] od_param;
+  
+  
+  if (do_overdispersion == 1) {
+    for (i in 1:N_admin_lev) {
+      od_param[i] = 1/inv_od_param[i];
+    }
+  }
   
-  if (do_infer_sd_eta == 1) {
+  if (size_sd_eta == 1) {
```

Comment:
Not quite sure I follow why this was changed to size_sd_eta... The do_infer_sd_eta parameter still exists and is specified in the config elsewhere

Reply:
Not sure either why this changed. In any case these are equivalent, and we will be deprecating this stan model version in the merge.